### PR TITLE
101 pep8 violation in the code

### LINKED ISSUE #101: Pep8 violation in the code
In some places, you have syntax like this:

```
if not xalign == None:
```

It's actually against Python coding standard -> https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or

Use either:
```
xalign is None
```
or
```
xalign is not None
```

Statement:
```
not xalign == None
```

Is a double whammy as it's using a double negation, and it's hard to follow when it's True and when it's False, imagine those cases:

```
not "" == None -> True
not None == None -> False
not 0 == None -> True
not 1 == None -> True
not False == None -> True
```

So perhaps simpler:
```
xalign is not None
```
would suffice?

### PR DESCRIPTION
**Pull request by Drachenfels**
Hi there,

This PR does two things:
1. c9a917aefb3c99fe980a64963debeca25143b6f7 - adds micro test suite that tests button, it detected one or two bugs with the current implementation
2. d40f8aaef3f32a6a3e86bb9a826e67fc87cf2bc3 - fixes detected errors and makes code somewhat more 'pythonic' that is the way a typical seasoned python programmer would write it (basically I run the code against pylint and black and if either of them complained I fixed that)

To run tests in your local env it's as easy as installing pytest (`pip install pytest`) and then running in shell `pytest -v`, output will be something like that:

```
platform linux -- Python 3.9.5, pytest-8.2.0, pluggy-1.5.0 -- ~/Envs/drachenfels-renpy-NQTR-System/bin/python
cachedir: .pytest_cache
rootdir: ~/Projects/drachenfels/renpy/NQTR-System
collected 12 items                                                                                                                           

tests/test_nqtr_button.py::test_button PASSED                                                                                          [  8%]
tests/test_nqtr_button.py::test_align_not_working_as_expected PASSED                                                                   [ 16%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[0-0-0-0_0] PASSED                                           [ 25%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[1-1-1-1] PASSED                                             [ 33%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[-1--1--1--1] PASSED                                         [ 41%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[1.1-1.2-1.1-1.2] PASSED                                     [ 50%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[-1.1--1.2--1.1--1.2] PASSED                                 [ 58%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[--0-0] PASSED                                               [ 66%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[False-False-0-0] PASSED                                     [ 75%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[None-None-0-0] PASSED                                       [ 83%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[xalign8-yalign8-0-0] PASSED                                 [ 91%]
tests/test_nqtr_button.py::test_button_aligns_accept_any_param_except_none[0-0-0-0_1] PASSED                                           [100%]
```

### CODE REVIEW

**Comment by Drachenfels** [comment]
I did not similar check anywhere else because I am not sure if you will be happy with those changes, but if yes, I can do more cleanups and fixes like that.

**Comment by BlackRam-oss** [comment]
> I did not similar check anywhere else because I am not sure if you will be happy with those changes, but if yes, I can do more cleanups and fixes like that.

Hi, I need to make sure that these changes don't change the logic of the tool, but otherwise it's fine.

If you want to make other changes that you think are necessary, just open another issue
